### ISSUE TITLE
request.validate() does return undefined

### ISSUE THREAD

**Issue by agtabesh** [adonisjs/adonis-validation-provider#74]
## Package version
5.0.0-preview

## Node.js and npm version
v12.14.1 and 6.13.4

## Sample Code (to reproduce the issue)
The code below does return `undefined`:
```
const postSchema = validator.compile(schema.create({
  title: schema.string(),
  body: schema.string(),
}))
  
const data = await request.validate({
  schema: postSchema,
})

console.log(data)
```
But simple `request.all()` returns all expected data from submitted form.

**Comment by thetutlage** [comment]
What's the output of `request.all()`?

**Comment by agtabesh** [comment]
@thetutlage the actual submitted value including `title` and `body` which both of them are `string`.

**Comment by thetutlage** [comment]
You can share the output of `request.all()` as it is?

**Comment by agtabesh** [comment]
@thetutlage here it is:
```
{
    "title": "title text",
    "body": "body text"
}
```

**Comment by thetutlage** [comment]
Weird!. Is it possible for you to share a repo that I can use to reproduce the issue?

**Comment by agtabesh** [comment]
Yes, here you are:
[https://github.com/agtabesh/request-validate-bug](https://github.com/agtabesh/request-validate-bug)

**Comment by thetutlage** [comment]
Thanks for reporting. Releasing the fix

**Comment by thetutlage** [comment]
Released as `6.2.7`. Running `npm i @adonisjs/validator@alpha` should fix the issue
